### PR TITLE
Fix scenario 1a

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

**Comment by vmoravec** [comment]
+1

File: scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-1a-ha-kvm-swift.yaml [hunk 1]
```diff
@@ -190,10 +190,12 @@ proposals:
       ceilometer-server:
       - cluster:services
       ceilometer-swift-proxy-middleware:
-      - cluster:services
+      - "@@controller1@@"
```
Comment (abel-navarro) [inline scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-1a-ha-kvm-swift.yaml]: this was already addressed in https://github.com/SUSE-Cloud/automation/pull/840.

**Comment by abel-navarro** [comment]
+1

**Comment by vmoravec** [comment]
Q: Do we already track the issue from https://github.com/SUSE-Cloud/automation/pull/840 somewhere, or is it matter of HA or upstream?

File: scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-1a-ha-kvm-swift.yaml (hunk 1 above)
Comment (vuntz) [inline scripts/scenarios/cloud6/qa/ssl-insecure/qa-scenario-1a-ha-kvm-swift.yaml]: Not for this file :/

**Comment by vuntz** [comment]
> Q: Do we already track the issue from #840 somewhere, or is it matter of HA or upstream?

Sorry, I don't understand the question :-) Can you rephrase?

**Comment by vmoravec** [comment]
I mean we know  `ceilometer-swift-proxy-middleware` doesn't work currently with cluster, is there a bug for this? I did not found anything for ceilometer in bsc..

**Comment by vuntz** [comment]
> I mean we know ceilometer-swift-proxy-middleware doesn't work currently with cluster, is there a bug for this?

Ah, thanks. No, there's no bug, and it's not really a bug, just a surprising behavior (which is why there's nothing in bugzilla).
